### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by nikgraf**
This removes the legacy RandomButton completely and replace it with the new one.

In addition I fixed the button which didn't propagate the props including `onClick` to the `Button` component.

### CODE REVIEW

File: plugins/react/frontend/components/PropForm/index.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
 import renderControls from '../../utils/renderControls';
-import RandomButton from '../common/RandomButton';
+import RandomButton from '../form/RandomButton';
```
Comment (mxstbr) [inline plugins/react/frontend/components/PropForm/index.js]: Since this is now shared, can we move it from `/form/` to `/common/`?

File: plugins/react/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ CarteBlanche is maintained by [@mxstbr](https://github.com/mxstbr), [@nikgraf](h
 
 ## to be released
 
+- changed; use the same RandomButton to randomize all the props
+- added; introduced a RefreshButton to reload the iFrame
```
Comment (mxstbr) [inline plugins/react/CHANGELOG.md]: Thanks, forgot to update the CHANGELOG!

**Comment by mxstbr** [comment]
Looks good, one question and CI fix then we can :shipit:

File: plugins/react/frontend/components/PropForm/index.js (hunk 1 above)
Comment (nikgraf) [inline plugins/react/frontend/components/PropForm/index.js]: Yep, will do this in a separate PR

Comment (nikgraf) [inline plugins/react/frontend/components/PropForm/index.js]: I mean we don't need form and just should move all components to `common`
